examples/lsm9ds0_simpletest.py

```diff
@@ -12,6 +12,17 @@
 i2c = busio.I2C(board.SCL, board.SDA)
 sensor = adafruit_lsm9ds0.LSM9DS0_I2C(i2c)
 
+#SPI connection:
+# from digitalio import DigitalInOut, Direction
+# spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
+# csag = DigitalInOut(board.D5)
+# csag.direction = Direction.OUTPUT
+# csag.value = True
+# csm = DigitalInOut(board.D6)
+# csm.direction = Direction.OUTPUT
+# csm.value = True
+# sensor = adafruit_lsm9ds0.LSM9DS0_SPI(spi, csag, csm)
+
```

Comment:
This example copy-pasted from LSM9DS1 is probably all wrong also. For one thing, the CS pins are taken in a different order for the LSM9DS0:
```python
def __init__(self, spi, xmcs, gcs):
```